source/loader/layers/sanitizer/sanitizer_common/sanitizer_stacktrace.cpp

```diff
@@ -93,10 +92,9 @@ void StackTrace::print() const {
         BacktraceInfo BI = BacktraceSymbols[i];
 
         // Skip runtime modules
-        if (!getContext()->interceptor->getOptions().Debug &&
```

Comment:
I think we should keep this, maybe by creating a base class of option that have the common options available for all sanitizers, and in this case, the "debug" switch. But that can be addressed later.

Reply:
I agree. Besides, I started to prefer to create a single option object for all sanitizers. 
We can discuss this later.